README.md

```diff
@@ -0,0 +1,112 @@
+Inheritance Shares Calculator
+
+This application calculates the inheritance shares for sons, daughters, and other family members based on the total inheritance amount and family structure.
+Features
+
+    Inheritance calculation for sons, daughters, wife, mother, and father.
+    Error handling for invalid input types.
+    Unit testing is included to ensure correctness of the logic.
+
+Requirements
+
+    Python 3.x
+    unittest for unit testing.
```

Comment:
These requirements are not enough, I mentioned in another PR that we need detail how someone can run this locally if he doesn't have the virtual environment and your requirements installed. e.g. unittest etc. Also attach a `requirements.txt` file with the required dependencies.

Reply:
@segullshairbutt What do you means, by creating a separate file i used to give details about it..?